api/src/pcapi/core/categories/subcategories_v2.py

```diff
@@ -23,7 +23,7 @@ class SearchGroups(Enum):
     CD_VINYLE_MUSIQUE_EN_LIGNE = "CD, vinyles, musique en ligne"
     CONCERTS_FESTIVALS = "Concerts & festivals"
     EVENEMENTS_EN_LIGNE = "Évènements en ligne"
-    FILMS_SERIES_CINEMA = "Films, séries, cinéma"
+    FILMS_SERIES_CINEMA = "Cinéma, films et séries"
```

Comment:
Cela nécessitera une réindexation totale des offres, car le "search group" est inclus dans la sérialisation d'une offre envoyée à Algolia (cf. code ci-dessous). Est-ce bien prévu ?

https://github.com/pass-culture/pass-culture-main/blob/48c63aeca68afab62dbff5ff008ec1b4215753ee/api/src/pcapi/core/search/backends/algolia.py#L563-L564

Reply:
C'est noté, c'est pas encore prévu.

Reply:
Oh la, oui, tu as raison. Effectivement, on indexe le nom du groupe, pas son libellé. Pas besoin de réindexer. Je me suis trompé.